### PR TITLE
supporting activemodel >= 5.0 #392

### PR DESCRIPTION
**Pull request by zkwentz**

### CODE REVIEW

**Comment by ryanswood** [comment]
Please merge this in. Gems that depend on Her and that are included in a Rails app are blocking upgrading to Rails 5.

File: her.gemspec [hunk 1]
```diff
@@ -23,8 +23,8 @@ Gem::Specification.new do |s|
   s.add_development_dependency "fivemat", "~> 1.2"
   s.add_development_dependency "json", "~> 1.8"
 
-  s.add_runtime_dependency "activemodel", ">= 3.0.0", "<= 4.3.0"
-  s.add_runtime_dependency "activesupport", ">= 3.0.0", "<= 4.3.0"
+  s.add_runtime_dependency "activemodel", ">= 3.0.0", "<= 5.0.0"
```
Comment (krishantaylor) [inline her.gemspec]: Shouldn't this be `< 5.1.0` so patches to rails 5 are compatible?

**Comment by zkwentz** [comment]
If anyone really wants to use this, I've forked and created a new gem: https://rubygems.org/gems/her5

**Comment by jkappers** [comment]
+1

**Comment by arturseletskiy** [comment]
+1

**Comment by edtjones** [comment]
hi @zkwentz @ryanswood @arturseletskiy @jkappers I've just got commit access to this repo and this will be my first merge! Will keep you posted here.

**Comment by edtjones** [comment]
Hi folks,

Rather than accept this PR I've written a slightly more permissive change to the gemspec. I'll close this one now.

Ed

**Pull request closed without merging**